iogt/templates/wagtailusers/groups/includes/formatted_permissions.html

```diff
@@ -41,6 +47,21 @@ <h2>{% trans "Object permissions" %}</h2>
                                 {{ content_perms_dict.delete.checkbox.tag }}
                             {% endif %}
                         </td>
+                        {% if custom_perms_exist %}
+                        <td>
+                            {% if content_perms_dict.custom %}
```

Comment:
Could this be a redundant check? If `custom_perms_exist` is true, does that not imply `content_perms_dict.custom` exists? If not, should we really just be checking that `content_perms_dict.custom` exists?

Reply:
@istride it's wagtail code. Do we need to change this?

Reply:
It would have been helpful to explain a little why this is the preferred solution. As I understand it, we are overriding this template due to a previous issue (which still stands), and we are merely synchronising this template with the one in Wagtail 2.15 (which has changed).

I find the code is questionable, and I wouldn't want us to be complacent about it because it has been copied from upstream. In this case, everything considered, I think it's ok to leave it as it is.

I am now wondering about other templates we might have overridden and whether we have a process for identifying them and synchronising them with any upstream changes.